File: newMProjects/newM/src/main/java/br/com/projeto/newM/cliente/Cliente.java
package br.com.projeto.newM.cliente;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.SequenceGenerator;
import javax.persistence.Table;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

@Entity
@Table(name = "cliente")
public class Cliente implements Serializable {

	private static final long serialVersionUID = 7559578928624446576L;

	@Id
	@SequenceGenerator(name = "cliente_seq", sequenceName = "cliente_seq", allocationSize = 1)
	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "cliente_seq")
	@Column(name = "cod_cliente", nullable = false, unique = true)
	private Long codCliente;

	@Column(name = "nome", nullable = false, length = 100)
	private String nome;

	@Temporal(TemporalType.TIMESTAMP)
	@Column(name = "nascimento", nullable = false)
	private Date nascimento;
	
	@Column(name = "cpf", nullable = false)
	private Long cpf;
	
	@Column(name = "celular", nullable = false)
	private Long celular;
	
	@Column(name= "email", nullable = false, length = 50)
	private String email;
	
	@Column(name = "endereco", nullable = false, length = 250)
	private String endereco;
	
	@Column(name = "observacao", length = 300)
	private String observacao;

	public Long getCodCliente() {
		return codCliente;
	}

	public void setCodCliente(Long codCliente) {
		this.codCliente = codCliente;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public Date getNascimento() {
		return nascimento;
	}

	public void setNascimento(Date nascimento) {
		this.nascimento = nascimento;
	}

	public Long getCpf() {
		return cpf;
	}

	public void setCpf(Long cpf) {
		this.cpf = cpf;
	}

	public Long getCelular() {
		return celular;
	}

	public void setCelular(Long celular) {
		this.celular = celular;
	}
	
	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

	public String getEndereco() {
		return endereco;
	}

	public void setEndereco(String endereco) {
		this.endereco = endereco;
	}

	public String getObservacao() {
		return observacao;
	}

	public void setObservacao(String observacao) {
		this.observacao = observacao;
	}	
}

File: newMProjects/newM_Front_End/client/js/app/models/Cliente.js
class Cliente {
    
    constructor(codCliente, nome, nascimento, cpf, celular, email, endereco, observacao) {        
        this._codCliente = codCliente;
        this._nome = nome;
        this._nascimento = new Date(nascimento.getTime());
        this._cpf = cpf;
        this._celular = celular;
        this._email = email;
        this._endereco = endereco;
        this._observacao = observacao;
        Object.freeze(this);
    }

    get codCliente() {
        return this._codCliente;
    }
    get nome() {
        return this._nome;
    } 
    get nascimento() {        
        return new Date(this._nascimento.getTime());
    }    
    get cpf() {
        return this._cpf;
    }
    get celular() {
        return this._celular;
    }
    get email() {
        return this._email;
    }
    get endereco() {
        return this._endereco;
    }
    get observacao() {
        return this._observacao;
    }
}